oslabs-beta/Kubiquity
# Minor style changes

WHAT DOES THIS PR DO? 
1. Updates code-styling to the FE components. 

HOW TO PULL THIS PR:
1. If you haven't yet installed Github's CLI, run `brew install gh` in your terminal. 
2. Once `gh` is installed, run `gh pr checkout 5`. You may be prompted to enter your GH credentials—do this! You also will be asked to choose a base repository—**CHOOSE oslabs-beta/Kubiquity!**

HOW TO TEST THIS PR:
1. In one terminal, run `npm install && npm run watch`. 
2. In a second terminal, run `npm start`. 
3. Verify that the app still works and looks the same as before. 

ADDITIONAL NOTES: 
1. Because `Error` is a reserved class in JS, I changed our `Error` component to `PodError`.
